wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -164,61 +179,48 @@ public Response create(@ApiParam(value = "The configuration to create the new wo
                   notes = "Composite key can be just workspace ID or in the " +
                           "username:workspace_name form, where username is optional (e.g :workspace_name is valid key too.")
     @ApiResponses({@ApiResponse(code = 200, message = "The response contains requested workspace entity"),
-                   @ApiResponse(code = 404, message = "The workspace by specified key does not exist"),
+                   @ApiResponse(code = 404, message = "The workspace with specified id does not exist"),
                    @ApiResponse(code = 403, message = "The user is not workspace owner"),
                    @ApiResponse(code = 500, message = "Internal server error occurred")})
-    public UsersWorkspaceDto getByKey(@ApiParam(value = "Composite key", examples = @Example({@ExampleProperty("workspace12345678"),
-                                                                                              @ExampleProperty("username:workspace_name"),
-                                                                                              @ExampleProperty(":workspace_name")})) @PathParam("key") String key)
-            throws NotFoundException,
-                   ServerException,
-                   ForbiddenException,
-                   BadRequestException {
+    public WorkspaceDto getByKey(@ApiParam(value = "Composite key",
+                                           examples = @Example({@ExampleProperty("workspace12345678"),
+                                                                @ExampleProperty("username:workspace_name"),
```

Comment:
It is not clear what is username for workspace. Maybe you should rewrite this with namespace?